core/templates/dev/head/pages/about/About.js

```diff
@@ -19,6 +19,19 @@
 oppia.controller('About', [
   '$scope', 'UrlInterpolationService',
   function($scope, UrlInterpolationService) {
+    $scope.listOfNames = ' Alex Kauffmann,' + ' Allison Barros,' +
```

Comment:
Use a list (defined as var) to store the names. Declare a separate $scope.xxx = listOfNames.join(', ') (though you may need to add some logic for the & bit).